src/h5web/toolbar/ComplexToolbar.tsx

```diff
@@ -0,0 +1,94 @@
+import { MdAspectRatio, MdGridOn } from 'react-icons/md';
+import ToggleBtn from './controls/ToggleBtn';
+import DomainSlider from './controls/DomainSlider/DomainSlider';
+import SnapshotButton from './controls/SnapshotButton';
+import Separator from './Separator';
+import Toolbar from './Toolbar';
+import ColorMapSelector from './controls/ColorMapSelector/ColorMapSelector';
+import ScaleSelector from './controls/ScaleSelector/ScaleSelector';
+import shallow from 'zustand/shallow';
+import { useComplexConfig } from '../vis-packs/core/complex/config';
+import {
+  ComplexVisType,
+  VIS_TYPE_SYMBOLS,
+} from '../vis-packs/core/complex/models';
+import Selector from './controls/Selector/Selector';
+import { useHeatmapConfig } from '../vis-packs/core/heatmap/config';
+
+function ComplexToolbar() {
+  const {
+    dataDomain,
+    customDomain,
+    setCustomDomain,
+    colorMap,
+    setColorMap,
+    scaleType,
+    setScaleType,
+    keepAspectRatio,
+    toggleAspectRatio,
+    showGrid,
+    toggleGrid,
+    invertColorMap,
+    toggleColorMapInversion,
+  } = useHeatmapConfig((state) => state, shallow);
```

Comment:
For now, the `ComplexToolbar` uses the same config as the `HeatmapToolbar` which means setting from the `Heatmap` will be persisted in the `Complex` and vice-versa.

We will wait for user feedback to see if this should be changed